{{cookiecutter.project_name}}/pyproject-flit621,trampolim.toml

```diff
@@ -0,0 +1,86 @@
+[build-system]
+{%- if cookiecutter.project_type == "trampolin" %}
```

Comment:
typo: `trampolim`

Reply:
Ahh, missed that. I'm thinking "trampoline without the e" in my head. That means the tests are actually testing flit621 again...

Reply:
Does trampolim allow an environment variable version override, like SETUPTOOLS_SCM_PRETEND_VERSION? If that was the case, we probably could drop version (and add it to dynamic), as flit can pick it up from the file (if I add flit621 to the if).

Reply:
```
AttributeError: module 'trampolim' has no attribute 'prepare_metadata_for_build_wheel'
```
Yes, now it's being tested...

Reply:
Oooh, I see `TRAMPOLIM_VCS_VERSION` in the source, nice, I think that's what I was asking about before.

Reply:
I think trampolim doesn't normalize names, so it's unhappy about "-" vs "_" in the project name.

Reply:
Ahh, that's not it, maybe it doesn't like src structure?

Reply:
> Ahh, missed that. I'm thinking "trampoline without the e" in my head. That means the tests are actually testing flit621 again...

Yeah, it's the portuguese word for trampoline.

> Ahh, that's not it, maybe it doesn't like src structure?

Yeah, it doesn't. It calculates the Python module name from the project name. I will probably add support for src layout in the future.